src/internal/easee/auth_test.go

```diff
@@ -0,0 +1,380 @@
+package easee //nolint
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/futurehomeno/cliffhanger/notification"
+	mockedstorage "github.com/futurehomeno/cliffhanger/test/mocks/storage"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/futurehomeno/edge-easee-adapter/internal/config"
+)
+
+type NotificationMock struct {
```

Comment:
Mocking part can go lower, after the testing methods. Also - we usually use mockery, and save mocks in one folder. But for a small thing like that I guess moving it would be enough.

Reply:
Right, I did it that way first, but there was circular dependencies because of the mocks in a single folder.
I would like to discuss that in more details but thought local mock might indeed work for now.